src/oncall/ui/static/js/loginsplash.js

```diff
@@ -0,0 +1,49 @@
+var oncall = {
```

Comment:
maybe change the name to `oncallLogin` or something to be more distinct from `oncall.js`